tools/bpf_inspect.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env drgn
+# SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause)
+#
+# Copyright (c) 2020 Facebook
+
+DESCRIPTION = """
+drgn script to list BPF programs or maps and their properties
+unavailable via kernel API.
+
+See https://github.com/osandov/drgn/ for more details on drgn.
```

Comment:
We probably don't need the mention to drgn with this tool in the drgn repository.